common/repo/GitRepo.java

```diff
@@ -98,7 +67,26 @@ private CommandResult runCommand(String command) {
     CommandResult result = new CommandResult();
     try {
       List<String> fullCommand = new ArrayList<>(gitCommandBase);
-      fullCommand.addAll(Arrays.asList(command.split(" ")));
+      if (!command.contains("\"")) {
```

Comment:
Why did you need all of this new code?

Reply:
It needs for `commit` command.
This command in our case has next syntax:
`commit -m "commit message"`
If we will split the command just using one space delimiter( command.split(" ") ) we get next parts:
[0] commit
[1] -m
[2] "commit
[3] message"

This code checks the command. If the command contains `"` character it groups all words between `"...."` characters in one part:
[0] commit
[1] -m
[2] commit message

Reply:
We start off with `"commit message"` in a separate string, then we combine it, then we work hard to split it again.

Instead, let's have another method `runCommand()` that gets a `List<String> command` that's already split, and for `commit` command, call it?
The current `runCommand(String command)` would trivially split based on spaces and call `runCommand(List<String> command)`

Reply:
Done.